docs/templates/layout.tmpl

```diff
@@ -9,10 +9,10 @@
     <!--[if lt IE 9]>
       <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
```

Comment:
This no longer works, for quite a long time now, maybe time to remove this as we're at it (and stop pretending IE8 is still a thing :wink:)

Reply:
Yep !